code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -115,6 +122,10 @@ var/global/list/datum/stack_recipe/plasteel_recipes = list ( \
 	sheettype = "plasteel"
 	stack_id = "plasteel"
 
+	radial_buildables = list(
+		/obj/structure/barricade/plasteel
```

Comment:
add crates and r-windows or it's just bleh

Reply:
this is the quick access menu for things you might actually want quick access to. i really doubt you need quick access to crates and reinforced window frames. if someone adds something important to plasteel later, like that non-folding plasteel barricade idea, they can slam it in here

Reply:
still having only one option is slightly defunct imo